MontyHallKata/Views/View.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using MontyHallKata.Models;
+using MontyHallKata.Models.Entity;
+using MontyHallKata.Views.InputOutput;
+
+namespace MontyHallKata.Views
+{
+    public class View
+    {
+        private readonly IInputOutput _inputOutputConsole;
+
+        public View(IInputOutput inputOutputConsole)
+        {
+            _inputOutputConsole = inputOutputConsole;
+        }
+
+        public int GetUserChoice()
+        {
+            _inputOutputConsole.PrintOutput(InputOutputMessages.ChoicePromptMessage);
+            
+            var choice = _inputOutputConsole.GetIntInput();
+            while (!InputValidation.IsValidChoice(choice))
+            {
+                _inputOutputConsole.PrintOutput(InputOutputMessages.InvalidInputMessage);
+                choice = _inputOutputConsole.GetIntInput();
+            }
+            return choice;
+        }
+
+        public int GetDoorSelectionFromUser()
+        {   
+            _inputOutputConsole.PrintOutput(InputOutputMessages.DoorSelectionPrompt);
+
+            var doorSelection = _inputOutputConsole.GetIntInput();
+            while (!InputValidation.IsValidDoorSelection(doorSelection))
+            {
+                _inputOutputConsole.PrintOutput(InputOutputMessages.InvalidInputMessage);
+                doorSelection = _inputOutputConsole.GetIntInput();
+            }
+            return doorSelection;
+        }
+
+        public void PrintDoors(Door[] doors)
+        {
+            var outputString = "";
```

Comment:
I notice that you mix and match static and dynamic typing a bit, for example this line here and line 25 of `InputOutputConsole`. Nothing major but another thing to be mindful of when it comes to consistency.